recipe/meta.yaml

```diff
@@ -21,33 +22,28 @@ source:
     sha256: 6f504b9282c83842c0601600f5e0663209b53b133125ec3334ff7a0bd0037ca6                               # [osx and x86_64]
   - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ osx_arm64_version }}-arm64-darwin-binary.tar.bz2  # [osx and arm64]
     sha256: 1d01fac2d9748f769c9246a0a11a2c011d7843337f8f06ca144f5a500e10c117                               # [osx and arm64]
-  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ win_64_version }}-x86-64-windows-binary.msi       # [win and x86_64]
-    sha256: 2cf132de1688b6e7d108128525446ae3de0fb7b98e3af016a1278593878be5a2                               # [win and x86_64]
-  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ win_64_version }}-source.tar.bz2                  # [win and x86_64]
+  - url: http://prdownloads.sourceforge.net/sbcl/sbcl-{{ version }}-source.tar.bz2                         # [win and x86_64]
     sha256: 8a932627b3f1d8e9618f1cdc225edcb002456804697e2c87d140683764a106d5                               # [win and x86_64]
     folder: sbcl-source                                                                                    # [win and x86_64]
     patches:
       - patches/0001-win-use-env-xargs.sh.patch  # [win]
       - patches/0002-win-update-flags-zstd.patch  # [win]
 
 build:
-  number: 3
-  script_env:
-    - MSI_FILE=sbcl-{{ win_64_version }}-x86-64-windows-binary.msi  # [win and x86_64]
+  number: 4
 
 requirements:
   build:
     - {{ compiler('c') }}  # [not win]
     - {{ stdlib('c') }}  # [not win]
-    # Current windows gcc toolchain (v5.3) segfaulta when building SBCL, need >=12
+    # Current windows gcc toolchain (v5.3) segfaults when building SBCL, need >=12
     # Using experimental msys2 toolchain for now
     - mingw-w64-ucrt-x86_64-gcc  # [win]
     - mingw-w64-ucrt-x86_64-gcc-libs  # [win]
     - mingw-w64-ucrt-x86_64-zstd  # [win]
     - make
   host:
-    # zstd is required to compile SBCL with 'compressed core saves' enabled
-    - zstd  # [win]
+    - zstd
```

Comment:
The `# [win]` selector should be kept, `zstd` is not required on the other platforms and leads to overlinking.

Reply:
It will be when building the other archs from source. It's easier for me to back-port these changes when I work on several PR at the same time, otherwise I become confused.

Reply:
Well, I should be more rigorous and use the correct and limited scope for each PR. Thank you